include/measurement_kit/swig.hpp

```diff
@@ -1,177 +0,0 @@
-// Part of Measurement Kit <https://measurement-kit.github.io/>.
```

Comment:
This API has been removed. The original idea was to wrap this using SWIG. But then I realised that, as long as we use sync APIs (like `nettest.hpp`) and manage threading in the target language (e.g. Java), we can quite easily wrap a much more rich API. So the point of this "swig" API was vanished.